### ISSUE TITLE
All system reset

### ISSUE THREAD

**Issue by E-m013** [sparticle999/SpaceCompany#471]
Wiped all saves on everything, probably because of update

**Comment by MachineGamer728** [comment]
yeah there isnt much to fix with this one, as you data is stored locally, so if you update or clear cookies it wipes it. Try and make a backup every couple of days in notepad or something

**Comment by JujuTheBee** [comment]
This happened to me too but it happened when i opened it from the main sparticle999 site
